lib/host.py

```diff
@@ -79,6 +79,15 @@ def execute_script(self, script_contents, shebang='sh', simple_output=True):
             finally:
                 self.ssh(['rm', '-f', script.name])
 
+    def ln(self, source, destination):
+        return self.ssh(['ln -s', source, destination])
+
+    def rm(self, path, flags=''):
+        return self.ssh(['rm', flags, path])
+
+    def mkdir(self, path, flags=''):
+        return self.ssh(['mkdir', flags, path])
```

Comment:
IMHO, it's better for user to have functions like:
```python
    def remove_file(self, path, force=False):
        args = ['rm', path]
        if force:
            args.append('-f')
        return self.ssh(args)
```

Reply:
I dropped this one.